### DISCUSSION TITLE
Credit card auto transfer

### DISCUSSION THREAD

**Discussion by Calinatus** [moneymanagerex/moneymanagerex#5913] [Q&A]
Hi all

My credit card final balance is made each 25th of each month. Whatever I have spent between the 25th of month X and the 25th of month Y will be taken from my bank account on the 3rd of each subsequent month. Is there any reason to put this into MMEX?

E.g.

24 May - I spend € 200 with credit card, balance is € 7,000 - € 200 = € 6,800
25 May - I spend € 400 with credit card, balance is € 6,800 - € 400 = € 6,400

3 June - Credit card is charged: € 200 from bank account; balance credit card is € 6,400 + € 200 = € 6,600

Any way to do this automatically?

Thanks!

**Comment by Macaulay1234** [comment]
Yes

**Comment by Macaulay1234** [comment]
> Hi all
> 
> My credit card final balance is made each 25th of each month. Whatever I have spent between the 25th of month X and the 25th of month Y will be taken from my bank account on the 3rd of each subsequent month. Is there any reason to put this into MMEX?
> 
> E.g.
> 
> 24 May - I spend € 200 with credit card, balance is € 7,000 - € 200 = € 6,800
> 25 May - I spend € 400 with credit card, balance is € 6,800 - € 400 = € 6,400
> 
> 3 June - Credit card is charged: € 200 from bank account; balance credit card is € 6,400 + € 200 = € 6,600
> 
> Any way to do this automatically?
> 
> Thanks!
